tooltips

pascal-pfeiffer opened the pull request:
adds tooltips to all experiment settings

closes #36

psinger commented on README.md:
```diff
@@ -106,12 +105,13 @@ With H2O LLM Studio, training your large language model is easy and intuitive.
 - **Mask Prompt Labels**: This option controls whether to mask the prompt labels during training and only train on the loss of the answer.
 - **Hyperparameters** such as learning rate, batch size, and number of epochs determine the training process.
-An overview of all parameters is given in the [parameter description](docs/parameters.md).
+Please consult the tooltips of each hyperparameter to learn more about them.
```
Let's also link to the folder here, not everyone checks the GUI.

psinger reviewed (changes requested):
Thanks!

The following tooltips are missing:

- Validation Dataframe
- Sync Batch Normalization
- Use Fsdp

Shall we remove the sync BN? Not too useful for Transformers I guess.

And shall we also have some sort of developer guide at some point that one needs to add this tooltip etc. But not too important at this point.

pascal-pfeiffer commented:
> Thanks!
> 
> The following tooltips are missing:
> 
> * Validation Dataframe
> * Sync Batch Normalization
> * Use Fsdp
> 
> Shall we remove the sync BN? Not too useful for Transformers I guess.
> 
> And shall we also have some sort of developer guide at some point that one needs to add this tooltip etc. But not too important at this point.


Thank you for the review, I removed BN sync and added the two missing tooltips. 
If a tooltip doesn't exist, it will just not be shown in the UI, so I don't think that it is super important to have a developer guide for this matter.

psinger reviewed (approved):
ty!
